src/utilities/input.tsx

```diff
@@ -13,6 +23,46 @@ type HelperTextProps = {
     validationStyle?: Validation;
 };
 
+type InputStyleGroup =
+    | 'base'
+    | 'width'
+    | 'height'
+    | 'element'
+    | 'focus'
+    | 'focusWithin'
+    | 'hover'
+    | 'disabled'
+    | 'readOnly';
+
+export const InputStylesLightTheme: Record<InputStyleGroup, string> = {
```

Comment:
@ryancarville I am wondering if it's really necessary to separate light from dark theme or if we just have `InputStyles` -> 
and each state base, hover, disabled etc... would include their base classes and dark prefixed ones.
Or what was your reasoning?

Reply:
I was trying to extract the specific styling based on their type or theme for more granular control/exporting. But can combined if desired.

Reply:
@ryancarville In this case, I think it's better to combine as we don't have a theme property on the inputs (yet). We now have some themed components in Fondue and it could become confusing. Take the `Toast` for example, in light theme it uses `base` which changes to gray on darkMode. But on dark theme, it uses mighty which on darkMode changes to mighty-inverse.

Reply:
Understood. I will combine them.